### PR TITLE
Feature/initial

### PR DESCRIPTION
**Pull request by annashamray**
Fixes https://github.com/VNG-Realisatie/gemma-zaken/issues/1622

### CODE REVIEW

File: src/contactmomenten/conf/api.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from vng_api_common.conf.api import *  # noqa - imports white-listed
+
```
Comment (joeribekker) [inline src/contactmomenten/conf/api.py:2]: Please add:

```
API_VERSION='1.0.0-beta'
```

File: src/contactmomenten/api/schema.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+te vragen.
+
+**Afhankelijkheden**
+
+Deze API is afhankelijk van:
+
+* Autorisaties API
+* Notificaties API
+* Klanten API
+* Zaken API *(optioneel)*
```
Comment (joeribekker) [inline src/contactmomenten/api/schema.py:18]: Please add:

```
    * Verzoeken API *(optioneel)*
    * Documenten API *(optioneel)*
```

File: src/contactmomenten/datamodel/models.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    )
+    bronorganisatie = RSINField(
+        help_text="Het RSIN van de Niet-natuurlijk persoon zijnde de "
+        "organisatie die de klantinteractie heeft gecreeerd. Dit moet een "
+        "geldig RSIN zijn van 9 nummers en voldoen aan "
+        "https://nl.wikipedia.org/wiki/Burgerservicenummer#11-proef"
+    )
+    klant = models.URLField(
+        max_length=1000,
+        blank=True,
```
Comment (joeribekker) [inline src/contactmomenten/datamodel/models.py:27]: I think it should say `null=True` here as well.

File: src/contactmomenten/datamodel/models.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        help_text=_("URL-referentie naar een KLANT (in Klanten API)"),
```
Comment (joeribekker) [inline src/contactmomenten/datamodel/models.py:28]:
```suggestion
        help_text=_("URL-referentie naar een KLANT (in Klanten API) indien de klantinteractie niet anoniem is."),
```

File: src/contactmomenten/conf/api.py (hunk 1 above)
Comment (annashamray) [reply src/contactmomenten/conf/api.py:2]: done

File: src/contactmomenten/datamodel/models.py (hunk 3 above)
Comment (joeribekker) [reply src/contactmomenten/datamodel/models.py:27]: ok.

**Pull request merged**